src/representations/calculator_base.hh

```diff
@@ -50,51 +50,70 @@ namespace rascal {
     //! type used to register the valid key and values of Hypers_t
     using ReferenceHypers_t = std::map<std::string, std::vector<std::string>>;
 
-    // TODO(felix) make sure it is not need anymore
-    using Dense_t = Eigen::Matrix<Precision_t, Eigen::Dynamic, Eigen::Dynamic,
-                                  Eigen::RowMajor>;
-    using InputData_t =
-        internal::InternallySortedKeyMap<std::vector<int>, Dense_t>;
-    using Data_t = std::vector<InputData_t>;
+    using Key_t = std::vector<int>;
 
-    RepresentationManagerBase() = default;
+    CalculatorBase() = default;
```

Comment:
Delete the standard constructor for safety, or do we need that somewhere?

Reply:
we need it.